app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -523,6 +527,23 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
         this.visualCompletenessQueue = visualCompletenessQueue
     }
 
+    private fun captureSnapshotTelemetryMetrics() = CoroutineScope(Dispatchers.IO).launch {
+        // PWA
+        val recentlyUsedPwaCount = components.core.webAppShortcutManager.recentlyUsedWebAppsCount(
```

Comment:
what a convenient call, is it just for telemetry?

Reply:
Yup.